clustersmgmt/v1/cloud_providers_client.go

```diff
@@ -100,6 +101,14 @@ func (r *CloudProvidersListRequest) Impersonate(user string) *CloudProvidersList
 	return r
 }
 
+// FetchRegions sets the value of the 'fetch_regions' parameter.
```

Comment:
FYI the param is actually called `fetchRegions`, correctly sent on L180, just the comments generated by metamodel are still inaccurate when overriding url param names with `@http` override.

(These CS APIs are not important for AMS anyway)